packages/web-components/src/components/va-checkbox/va-checkbox.tsx

```diff
@@ -219,7 +219,7 @@ export class VaCheckbox {
       'va-checkbox__container--tile--checked': tile && checked,
     });
     const descriptionClass = classnames({
-      'usa-legend': true,
+      'vads-u-margin-top--1p5': true,
```

Comment:
I'm not sure if this is the correct margin.

Reply:
I think that's a good guess (I don't see anything in [Figma](https://www.figma.com/design/afurtw4iqQe6y4gXfNfkkk/VADS-Component-Library?node-id=137-17116&p=f&t=UjYhARqFoBAeg7Nb-0) to validate) but regardless of the margin size used, I'm wondering if we even want to use vads utility classes like this? Or should the style be added directly to the component's stylesheet? 

My instinct is that we should maintain component style encapsulation as much as possible. I know we've tried in the past to keep Formation styles outside of components for that reason and this feels like the same thing.

![Screenshot 2025-03-12 at 10 08 46 AM](https://github.com/user-attachments/assets/a730ea6d-2bc3-4f26-823b-41d82245ba71)

Reply:
@jamigibbs - good call!